docs/uml/macros.iuml

```diff
@@ -392,17 +392,15 @@ skinparam defaultMonospacedFontName Andale Mono
     }
 !endif
 
-!ifdef SHOW_SOURCE_CONTRACT
+!ifdef SHOW_PROCESSED_SOURCE
```

Comment:
Will `processedSource` not have a name field? I believe it still has one in the schema so I wanted to check in on this!

Reply:
Hm I don't think it needs one (not sure what it'd be useful for on its own)